src/software/gui/drawing/robot.cpp

```diff
@@ -13,11 +17,11 @@ void drawRobotVelocity(QGraphicsScene* scene, const Robot& robot, const QColor&
     pen.setCapStyle(Qt::PenCapStyle::RoundCap);
     pen.setCosmetic(true);
 
-    drawSegment(scene, Segment(robot.position(), robot.position() + robot.velocity()),
-                pen);
+    drawSegment(scene, Segment(position, position + velocity), pen);
 }
 
-void drawRobotPosition(QGraphicsScene* scene, const Robot& robot, const QColor& color)
+void drawRobotPosition(QGraphicsScene* scene, const Point& position,
+                       const Angle& orientation, const QColor& color)
```

Comment:
Could we rename to `drawRobotAtPosition`?

Reply:
done